Hunting Queries/ASimRegistry/CrashdumpdisabledonhostASIM.yaml

```diff
@@ -0,0 +1,25 @@
+id: 6cb193f3-7c6d-4b53-9153-49a09be830d7
+name: Crash dump disabled on host ASIM
+description: |
+  'This detection looks the prevention of crash dumps being created. This can be used to limit reporting by malware, look for suspicious processes setting this registry key.'
+requiredDataConnectors: []
+tactics:
+  - DefenseEvasion
+relevantTechniques:
+  - T1070
+query: |
+    imRegistry
+    | where RegistryKey == "SYSTEM\\CurrentControlSet\\Control\\CrashControl"
```

Comment:
imRegistry uses the MDE style registry names. See [here](https://docs.microsoft.com/en-us/azure/sentinel/registry-event-normalization-schema#root-keys)